docs/expressions.html

```diff
@@ -408,7 +408,7 @@ <h2 class="item_title">
 <li><code>DD</code> - day of month, zero padded 0-31</li>
 <li><code>h</code> - hour of the day 1-12</li>
 <li><code>hh</code> - hour of the day 01-12</li>
-<li><code>tt</code> - twenty four hour of the day 01-23</li>
+<li><code>tt</code> - twenty four hour of the day 01-24</li>
```

Comment:
shouldn't this be 00-23 ?

Reply:
Ack, I guess so, though both are accepted, so different off by 1. Will update to 0-23:
  https://en.wikipedia.org/wiki/24-hour_clock#Midnight_00:00_and_24:00